app/assets/stylesheets/image_x.scss

```diff
@@ -80,27 +80,32 @@ $image-thumb-slider-border-color: #000;
   float: left;
   margin: 0 5px 0 0;
   padding: 0;
+  img {
+      width:100%;
```

Comment:
Line should be indented 4 spaces, but was indented 6 spaces<br>Colon after property should be followed by one space